indico/modules/events/registration/controllers/management/reglists.py

```diff
@@ -37,6 +37,8 @@
 from indico.modules.events.registration.util import (get_event_section_data, make_registration_form,
                                                      create_registration, generate_csv_from_registrations)
 from indico.modules.payment import event_settings
```

Comment:
use whatever is more common. eventually we should rename this stuff where it's defined